app/mailers/admin_mailer.rb

```diff
@@ -0,0 +1,14 @@
+class AdminMailer < ApplicationMailer
+  default from: 'remote-flow@mail.com'
+
+  def delete_channel_email(channel)
```

Comment:
avoid name duplication in methods
`AdminMailer.delete_channel_email` -> `AdminMailer` already gives a clue that it's a mailer, and method is to specify event